app/controllers/schools/add_participants_controller.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module Schools
+  class AddParticipantsController < ::Schools::BaseController
+    FORM_SESSION_KEY = :add_participant_form
+    FORM_PARAM_KEY = :schools_add_participant_form
+
+    skip_after_action :verify_authorized
+    before_action :ensure_form_present, except: :start
+    before_action :set_school_cohort
+
+    helper_method :add_participant_form
+
+    def start
+      session.delete(FORM_SESSION_KEY)
+      session[FORM_SESSION_KEY] = {
+        school_cohort_id: @school_cohort.id,
+      }
+      redirect_to action: :show, step: :type
+    end
+
+    def show
+      render current_step
+    end
+
+    def update
+      if add_participant_form.valid?(current_step)
+        add_participant_form.record_completed_step current_step
+        store_form_in_session
+        redirect_to action: :show, step: step_param(add_participant_form.next_step(current_step))
+      else
+        render current_step
+      end
+    end
+
+    def complete
+      @participant = add_participant_form.save!
```

Comment:
So this is actually a `participant_profile`, right? I think I would like to stick to naming convention of 
`participant` = `user who is a mentor or ect`
`participant_profile` = `ect_profile or mentor_profile`, at least until we work out a neater way of handling them.